2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,60 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let recipes = {
+  carbonara: {
+    title: "Carbonara",
+    serves: 2,
+    ingredients: ["Spaghetti", "Guanciale", "Egg", "Cheese"],
+  },
+  pizza: {
+    title: "Pizza",
+    serves: 1,
+    ingredients: [
+      "Flour",
+      "Salt",
+      "Olive oil",
+      "Mozzarella",
+      "Passata",
+      "Parmesan",
+    ],
+  },
+  tiramisu: {
+    title: "Tiramisu",
+    serves: 3,
+    ingredients: ["Mascarpone", "Egg", "Savoiardi", "Sugar", "Caffè"],
+  },
+  amatriciana: {
+    title: "Amatriciana",
+    serves: 2,
+    ingredients: [
+      "Spaghetti",
+      "passata",
+      "Guanciale",
+      "Cheese",
+      "Salt",
+      "Oil",
+      "White wine",
+    ],
+  },
+  steakBroccoli: {
+    title: "Steak and Broccoli",
+    serves: 2,
+    ingredients: [
+      "Wholegrain",
+      "Chopped sushi ginger",
+      "Spring onions",
+      "Broccoli",
+      "Fillet steak",
+    ],
+  },
+};
+for (let recipe in recipes){
+  console.log(recipes[recipe].title);
```

Comment:
You use `recipes[recipe]` multiple times. I usually suggest introducing a variable to reduce code duplication. But in this case - would `for-of` loop work better?